.github/workflows/ubuntu.yml

```diff
@@ -92,6 +92,17 @@ jobs:
             otp-version: '24'
             rebar3-version: '3.15'
             os: 'ubuntu-20.04'
+          - elixir-version: 'v1.11.0'
+            otp-version: '22.3.4.2'
+            os: 'ubuntu-20.04'
+            version-type: 'strict'
+          - elixir-version: '1.10.3'
+            otp-version: '22.3.4.1'
+            os: 'ubuntu-20.04'
+            version-type: 'strict'
+          - elixir-version: '1.10.3'
```

Comment:
Same as before, without 'strict'. This proves strict isn't always required in non-semver cases.